Dockerfile

```diff
@@ -3,31 +3,11 @@
 # Optional requirements:
 #   nvidia-docker2 (https://github.com/NVIDIA/nvidia-docker)
 #
-# Build:
-#   docker build -t core3d/danesfield .
+# To run with CUDA support, ensure that nvidia-docker2 is installed on the host
 #
-# Run:
-#   docker run \
-#     -i -t --rm \
```

Comment:
It's not clear why they want to take out this documentation.  I suggested we leave this in.